### ISSUE TITLE
rule 'no-any'  with severinity 'warn' failed linting with error 'errors found in the listed files'

### ISSUE THREAD

**Issue by vasinkevych** [palantir/tslint#3501]
### Bug Report
When I try to configure rule'no-any' as a warning, it failed linting with an error.  Can be more useful to use it an option as the warning without errors( because it is a cause of failing build on CI).  
- __TSLint version__:
- __TypeScript version__:
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
function doSmth(a: any): any { 
    return a
}
```

with `tslint.json` configuration:

```json
    "no-any": {
      "severity": "warning",
      "options":[true]
    }

```

#### Actual behavior
 shows all any as warnings but  failed with error **errors found in the listed files.**
#### Expected behavior
Should not fail with warnings

**Comment by ajafff** [comment]
I cannot reproduce this.
The additional output comes from a different tool. Are you using `ng lint`?

**Comment by vasinkevych** [comment]
yes,  I use ng lint.
